File: MODUL 6/CLASS/PRAK604-2410817110003-FerianDestaAshadi.c
#include <stdio.h>
#include <string.h>
int main() {
    char kode[101], pesan[101];
    int star = 0, hash = 0;

    printf("");
    fgets(kode, sizeof(kode), stdin);
    printf("");
    fgets(pesan, sizeof(pesan), stdin);

    kode[strcspn(kode, "\n")] = '\0';
    pesan[strcspn(pesan, "\n")] = '\0';

    if (strlen(kode) != strlen(pesan)) {
        printf("Panjang kalimat berbeda, pesan palsu\n");
        return 0;
    }
    for (int i = 0; i < strlen(kode); i++) {
        if (kode[i] == ' ' && pesan[i] == ' ') {
            printf(" ");
        } else if (kode[i] == ' ' || pesan[i] == ' ') {
            printf("#");
            hash++;
        } else if (kode[i] == pesan[i]) {
            printf("*");
            star++;
        } else {
            printf("#");
            hash++;
        }
    }
    printf("\n");
    printf("* = %d\n", star);
    printf("# = %d\n", hash);

    if (star >= hash) {
        printf("Pesan Asli\n");
    } else {
        printf("Pesan Palsu\n");
    }
    return 0;
}

File: MODUL 6/CLASS/PRAK604-2410817110003-FerianDestaAshadi.py
kode = input("")
pesan = input("")

if len(kode) != len(pesan):
    print("Panjang kalimat berbeda, pesan palsu")
else:
    star = 0
    hash_count = 0
    
    result = []
    for k, p in zip(kode, pesan):
        if k == ' ' and p == ' ':
            result.append(' ')
        elif k == ' ' or p == ' ':
            result.append('#')
            hash_count += 1
        elif k == p:
            result.append('*')
            star += 1
        else:
            result.append('#')
            hash_count += 1
    print("".join(result))
    print(f"* = {star}")
    print(f"# = {hash_count}")
    if star >= hash_count:
        print("Pesan Asli")
    else:
        print("Pesan Palsu")